app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    file_name = command.split()[1]
```

Comment:
Consider using a more descriptive variable name instead of `file_name` to better convey its purpose, as per the checklist item: 'Use descriptive and correct variable names.'